src/commands/start.cr

```diff
@@ -8,10 +8,16 @@ module Mint
                   description: "Auto formats the source files when running development server.",
                   default: false,
                   required: false
+      define_flag port : Int32,
+                  description: "Change the port to serve the application on. (Default: 3000)",
+                  long: port,
+                  short: p,
+                  default: 3000_i32,
+                  required: false
```

Comment:
Also would be awesome if it respected the `PORT` and `HOST` environment variables.